src/components/shared/DetailsWidget/AmountDetail/AmountDetail.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from 'react';
+
+import { Token } from '~types';
+import TokenIcon from '~shared/TokenIcon';
+
+import styles from './AmountDetail.css';
+
+const displayName = 'DetailsWidget.AmountDetail';
+
+interface AmountDetailProps {
+  amount: ReactNode; // Numeral component
+  symbol: ReactNode; // span
+  token?: Token;
+}
+
+const AmountDetail = ({ amount, symbol, token }: AmountDetailProps) => (
```

Comment:
`ActionAmount` ? Although my suggestion is also not very descriptive

Reply:
I like the inclusion of "Detail" since it forms part of the "DetailsWidget" -> i.e. using "Detail" to describe a kind of thing (a row in the details widget), of which this is the "Amount" version